test/e2e/argo-workflows/cluster-agent.yaml

```diff
@@ -631,17 +630,6 @@ spec:
     - - name: no-more-nginx
         template: no-more-metrics-nginx
 
-  - name: query-job
-    activeDeadlineSeconds: 120
-    inputs:
-      parameters:
-      - name: manifest
-    resource:
-      action: create
-      successCondition: status.succeeded > 0
-      failureCondition: status.failed > 10
-      manifest: "{{inputs.parameters.manifest}}"
-
```

Comment:
@JulienBalestra This step also exists in the `agent` workflow. Do you know what it's for?

Reply:
@devonboyer this can be removed indeed